Update application-gateway-faq.md

ChrisLopezUK opened the pull request:
Added a small section on Connection Draining on request of a customer.

Q. Does changing backend pool members cause disruption?

azurecla commented:
Hi __@SeanFeldman__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

PRmerger commented:
@SeanFeldman : Thanks for your contribution to the Azure documentation! The author, @sethmanheim, has been notified to review your proposed change.

jtaubensee commented:
@sethmanheim - Can you add this to MSDN as well?

SeanFeldman commented:
MSDN: 
https://msdn.microsoft.com/en-us/library/microsoft.servicebus.messaging.messagereceiver.completeasync.aspx
https://msdn.microsoft.com/en-us/library/microsoft.servicebus.messaging.messagereceiver.completebatchasync.aspx

sethmanheim commented:
Sure will @jtaubensee. Thanks for the links, @SeanFeldman.

sethmanheim commented:
Actually @jtaubensee, for the .NET reference those changes will need to be made in the triple slash comments in the sources.

jtaubensee commented:
@sethmanheim - We can go ahead and merge this. I have a code review out for the source/msdn changes.

jtaubensee commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

SeanFeldman commented:
@jtaubensee 

![image](https://cloud.githubusercontent.com/assets/1309622/19699570/0c5be808-9ab1-11e6-90ed-50b782fa5b42.png)

sethmanheim commented:
I have the power... #sign-off

Commit:
Update application-gateway-faq.md

Added a small section on Connection Draining on request of a customer.

Q. Does changing backend pool members cause disruption?

vhorne reviewed (changes requested):
See Conversation for suggested wording.

ChrisLopezUK reviewed (commented):
Perfect. :)

Thank you!

Commit:
Update application-gateway-faq.md
